core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-category-list/dot-category-field-category-list.component.html

```diff
@@ -0,0 +1,40 @@
+<div class="w-full category-list__header">Root</div>
+<div
+    [ngClass]="{ 'no-overflow-x-yet': emptyColumns().length }"
+    class="flex-1 category-list__category-list">
+    @for (column of categories(); let index = $index; track index) {
+    <!--dynamic columns-->
+    <div #categoryColumn class="category-list__category-column" data-testId="category-column">
+        @for (item of column; track item.inode) {
+        <div
+            data-testId="category-item"
+            class="flex align-content-center align-items-center category-list__item"
+            [ngClass]="{ 'category-list__item--selected': item.checked }"
+            (click)="itemClicked.emit({ index, item })">
```

Comment:
@oidacra Here you can use `itemClicked.emit({ $index, item })` and avoid `let index = $index;`.